projects/packages/sync/src/modules/class-woocommerce-hpos-orders.php

```diff
@@ -303,7 +303,15 @@ private function filter_order_data( $order_object ) {
 		}
 		$filtered_order_data['status'] = $this->get_wc_order_status_with_prefix( $filtered_order_data['status'] );
 
-		return $filtered_order_data;
+		/**
+		 * Filter the order data before syncing.
+		 *
+		 * @since x.x.x
```

Comment:
```suggestion
		 * @since $$next-version$$
```
[Package version annotations](https://github.com/Automattic/jetpack/blob/c94dd7e291080ead12f4a94252075592668bca65/projects/packages/README.md#package-version-annotations)